### PR TITLE
Add jhb's Wireguard entry

### PR DESCRIPTION
**Pull request by Jehops**

### CODE REVIEW

**Review by pauamma — APPROVED** [review]

File: 2022q1/freebsd-foundation.adoc [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+==== OS Improvements
+
+During the first quarter of 2022, 372 src, 41 ports, and 16 doc tree
+commits were made that identified The FreeBSD Foundation as a sponsor.
+# This represents 16, 0.4, and 5% of the total number of commits in each repository.
+
+You can read about Foundation-sponsored projects in individual quarterly report
+entries:
+
+- Crypto changes for WireGurard
```
Comment (pauamma) [inline 2022q1/freebsd-foundation.adoc:57]:
```suggestion
- Crypto changes for WireGuard
```

File: 2022q1/freebsd-foundation.adoc [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+Tracing) provider to work with version 3 of CTF.
+
+==== FreeBSD on the Framework Laptop
+
+Two Foundation staff members, Ed Maste and Mark Johnston, as well as a few developers and community members now each have access to Framework laptops, which are designed to make hardware upgrades, repairs, and customizations straightforward for the average user.  The goal of this work is to ensure that the experience running FreeBSD on the laptops matches the stability that FreeBSD users expect.
+
+Recent improvements and fixes include:
+ making audio switch appropriately between speakers and the headphone jack when headphones are plugged in or unpluged
+A fix for bug 259230, which would cause a Framework laptop to reboot or power off when the touchpad was used.
+Adding the Tempo Semiconductor 92HD95B HDA codec ID
```
Comment (pauamma) [inline 2022q1/freebsd-foundation.adoc:95]: Make that a bullet list using "* "?

**Review by pauamma — APPROVED** [review]

**Comment by pauamma** [comment]
*glares at Github* Approving the crypto changes, requesting changes to the rest.

**Comment by pauamma** [comment]
I still see both, apparently identical to earlier versions.

**Comment by Jehops** [comment]
The problem with multiple entries in the same pull request was because I foolishly pushed the Foundation entry commit to the same branch that I used to submit the wireguard pull request.

The force push was because I updated my main branch from the main branch of github.com/freebsd/freebsd-quartery, then rebased my local branch for this pull request on top of the updated main.

I'll fix this branch to only include the wireguard changes, then submit a new pull request.

**Comment by pauamma** [comment]
> I'll fix this branch to only include the wireguard changes, then submit a new pull request.

Thanks.

**Pull request merged**